packages/react/src/components/Slider/Slider.tsx

```diff
@@ -379,12 +531,11 @@ export default class Slider extends PureComponent {
       return;
     }
 
-    const which = Number.parseInt(evt.which);
     let delta = 0;
-    if (matches(which, [keys.ArrowDown, keys.ArrowLeft])) {
-      delta = -this.props.step;
-    } else if (matches(which, [keys.ArrowUp, keys.ArrowRight])) {
-      delta = this.props.step;
+    if (matches(evt.which, [keys.ArrowDown, keys.ArrowLeft])) {
+      delta = -(this.props.step ?? Slider.defaultProps.step);
```

Comment:
@sierrawetmore Sorry if I'm being dense here on these default props - why is this needed?

What error is given if this is reverted to it's original version without the nullish coalescing operator providing the default as a fallback?